src/lib/components/lines-and-dots/svg/text.svelte

```diff
@@ -48,36 +36,27 @@
     strokeWidth={backdropHeight}
   />
 {/if}
-{#if showIcon && textAnchor === 'start'}
+{#if showIcon}
   <Icon
     name={icon}
-    x={x - offset}
+    x={textAnchor === 'end' ? x - textWidth - backdropHeight : x}
     y={y - 8}
```

Comment:
One thing to try is instead of this terrible math I did for y, use text-anchor="middle"

Reply:
We want the icon to show up at the start of the text though, so we need to do a bit of math to make that happen when `textAnchor === 'end'`.

Reply:
Like centering the y position without needing to do the y - 8

Reply:
Ah I see what you're saying, but the `text-anchor` attribute is for `<text/>` elements and that icon is not in one (it's just an `<svg/>` next to one).

Reply:
🤦 My bad. I thought it was text not the icon